horse/web/recommendations.py

```diff
@@ -2,18 +2,20 @@
 from flask_restful import Resource, Api
 
 from horse.recommendation.dummy_service import DummyRecommendationService
+from horse.web.schemas.movie import MovieSchema
```

Comment:
I think we can just import `movies_schema` and not have to re-instantiate it here